editor/editor.cpp

```diff
@@ -48,7 +49,29 @@ int main() {
         Engine::getSoundManager()->getSound("helloWorld")->play();
     }});
 
-    Engine::init([]{
+    if (!glfwInit()) {
```

Comment:
Is it possible to move this back into the engine and put anything related to GLFW behind a compile flag?